app/resources/qml.qrc

```diff
@@ -1,5 +1,7 @@
 <RCC>
-	<qresource prefix="/">
-		<file>main.qml</file>
-	</qresource>
+    <qresource prefix="/">
+        <file>main.qml</file>
+        <file>RosterPage.qml</file>
+        <file>LoginPage.qml</file>
+    </qresource>
```

Comment:
I think Qt Creator ignores indentation in .qrc files. I can fix it manually if you want.